### ISSUE TITLE
Expected newline after comma in array declarations 

### ISSUE THREAD

**Issue by AndreyUtka** [stylelint/stylelint#1190]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Expected newline after "," in a multi-line list for array declaration 
Maybe It's bug. In last version of 5.2.1 this does not occur. 
> Which rule, if any, is this issue related to?

 `value-list-comma-newline-after`

> What CSS is needed to reproduce this issue?

```scss
$grid-breakpoints: (
    (xs),
    (sm, 768px),
    (md, 992px),
    (lg, 1200px)
) !default;
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
        "value-list-comma-newline-after": "always-multi-line",
        "value-list-comma-space-after": "always-single-line",
        "value-list-comma-space-before": "never"
  }
}
```

> Which version of stylelint are you using?

`6.2.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

https://github.com/vieron/stylelint-webpack-plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS maps

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
Expected newline after "," in a multi-line list   value-list-comma-newline-after
```

**Comment by jeddy3** [comment]
@AndreyUtka Thanks again for another nice report :)

Unlike the other issue, it's not clearly a regression as not much has changed in this rule between `5.2` and `6.2`. Anyhow, there's a [pending PR](https://github.com/stylelint/stylelint/pull/1191) to ignore SCSS maps going forward, and so that will nuke this erroneous warning for you.

**Comment by AndreyUtka** [comment]
@jeddy3 yep) thanks a lot)
